Tutorials/useful_resources.md

```diff
@@ -0,0 +1,96 @@
+layout: default
+title: Useful Resources
+parent: Tutorials
+nav_order: 10
+mathjax: true
+---
+**Last Updated:** 2021-05-10
+
+🚧 Work In Progress
+{: .label .label-yellow}
+
+# Useful Resources to Get Started
+These are some links to different tools and resources on different subjects, languages, tools and
+more that can be useful for your work:
+
+## Python:
+#### MIT Intro to CS Course with Python:
+https://www.edx.org/course/introduction-to-computer-science-and-programming-7
+#### Kaggle – Quick Python Tutorial:
+https://www.kaggle.com/learn/python
+#### Style Guide for Python:
+https://pep8.org/
+#### TutorialsPoint Python Reference:
+https://www.tutorialspoint.com/python/index.htm
+
+## Numpy:
+
+#### Stanford CS231n – Numpy Tutorial:
+https://cs231n.github.io/python-numpy-tutorial/
+#### TutorialsPoint Numpy Reference:
+https://www.tutorialspoint.com/numpy/index.htm
+#### Linkedin Learning (Available with UW NetID):
+https://www.linkedin.com/learning/numpy-data-science-essential-training/welcome?u=67682169
+#### Numpy Cheat Sheet:
+https://www.kaggle.com/ravichaubey1506/essential-numpy
+
+## Pytorch:
+#### Official Tutorials:
+https://pytorch.org/tutorials/
+#### Pytorch Beginner Series – Official Youtube Videos:
+https://www.youtube.com/playlist?list=PL_lsbAsL_o2CTlGHgMxNrKhzP97BaG9ZN
+#### TutorialsPoint Pytorch Reference:
+https://www.tutorialspoint.com/pytorch/
+#### Udacity Course on Pytorch:
+https://www.udacity.com/course/deep-learning-pytorch--ud188
+#### Notebook-Heavy Pytorch Tutorial:
+https://github.com/MorvanZhou/PyTorch-Tutorial
+#### Large Reference List for Pytorch:
+https://github.com/ritchieng/the-incredible-pytorch
+
+## OpenAI Gym:
+#### Official Documentation:
+https://gym.openai.com/docs/
+#### Paperspace – Beginner Tutorial:
+https://blog.paperspace.com/getting-started-with-openai-gym/
+
+## Reinforcement Learning:
+#### Charles Isbell – Udacity RL Course:
+https://www.udacity.com/course/reinforcement-learning--ud600
+#### David Silver RL Youtube:
+https://www.youtube.com/playlist?list=PLqYmG7hTraZDM-OYHWgPebj2MfCFzFObQ
+#### CS234 – Stanford RL Course:
+http://web.stanford.edu/class/cs234/index.html
+
+https://www.youtube.com/playlist?list=PLoROMvodv4rOSOPzutgyCTapiGlY2Nd8u
+#### Byron Boots RL Class Notes:
+https://homes.cs.washington.edu/~bboots/RL-Fall2020/
+#### RL Book by Sutton & Barto:
+http://incompleteideas.net/book/the-book.html
+#### OpenAI – Spinning Up:
+https://spinningup.openai.com/en/latest/
+#### Pytorch Implementations of RL Algorithms:
+https://github.com/MishaLaskin/torchingup
+
+## Training/Configuration Frameworks:
+#### fast.ai:
+https://docs.fast.ai/
+#### Pytorch Ignite:
+https://github.com/pytorch/ignite
+#### Pytorch Lightning:
+https://github.com/PyTorchLightning/pytorch-lightning
+
+https://www.youtube.com/playlist?list=PLaMu-SDt_RB5NUm67hU2pdE75j6KaIOv2
+
+https://www.youtube.com/watch?v=w10WrRA-6uI
+#### Pytorch Lightning Bolts:
+https://lightning-bolts.readthedocs.io/en/latest/yuo
```

Comment:
Need to see if these are actually maintained or not. But could be good.

Reply:
Turns out these were not maintained because the URLs are dead

Reply:
https://lightning-bolts.readthedocs.io/en/latest/